app/src/main/java/com/seafile/seadroid2/SeafConnection.java

```diff
@@ -1032,6 +1032,21 @@ private String uploadBlocksCommon(String link, String repoID, String dir,
         }
     }
 
+    public void createNewLib(String libName, String description, String password) throws SeafException {
```

Comment:
we use `repo` internally to describe library, so better to refactor those names.